highfidelity/hifi
# Fix entity list refresh when entity deleted

Fix the entity list not updating to remove any entity deleted via either keyboard Delete, lifetime elapsing, or external deleteEntity call.

Fixes: https://highfidelity.manuscript.com/f/cases/14127/

Test Plan:
- Enter Interface in desktop mode
- Open Create and create any entity
- Shift to the List tab and find the selected entity in the list
- Select the entity again to focus in world away from the Create window
- Press Delete key and verify the deleted entity is automatically removed from the list
- Create another entity and copy it's ID in the Properties tab
- Find the selected entity in the List tab
- Open Developer menu -> Console and enter the following line and press enter:
Entities.deleteEntity(XXX);
where XXX is the ID you copied from the Properties tab above
- Verify the deleted entity is automatically removed from the list
- Create another entity and set it's lifetime in the Properties tab to 30
- Find the selected entity in the List tab
- Verify that after the lifetime elapses and the entity is deleted that it is automatically removed from the list
